integrated_channels/sap_success_factors/models.py

```diff
@@ -143,3 +155,74 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class SAPSuccessFactorsLearnerCompletionPayload(object):
```

Comment:
I agree with @brittneyexline  - I think the `LearnerDataTransmissionAudit` is sufficient.  Fewer classes is cleaner.

37299d0 moves the payload generation functionality I had in `SAPSuccessFactorsLearnerCompletionPayload` into a new `payload()` method.

It also overrides `__init__` to allow some fields to be pulled from optional `EnterpriseCourseEnrollment` and `GeneratedCertificate` objects.